oppijanumerorekisteri-service/src/main/resources/db/migration/V20200206110726043__huoltajasuhde_aikaleimat_idx.sql

```diff
@@ -0,0 +1,5 @@
+alter table henkilo_huoltaja_suhde add column created timestamp without time zone;
+alter table henkilo_huoltaja_suhde add column updated timestamp without time zone;
+
```

Comment:
Voisiko näissä olla NOT NULL -rajoitus? Ne kuitenkin asetetaan luodessa/päivittäessä.

Reply:
nyt olisi